### PR TITLE
Draft version of FBA.

### PR DESCRIPTION
**Pull request by yaochaorui**

### CODE REVIEW

File: configs/mattors/fba/baseline.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
+custom_imports = dict(
+    imports=['mmedit.core.optimizer.radam'], allow_failed_imports=False)
```
Comment (innerlee) [inline configs/mattors/fba/baseline.py:2]: Unnecessary imports

File: configs/mattors/fba/baseline.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+# dataset settings
+dataset_type = 'AdobeComp1kDataset'
+data_root = './data/adobe_composition-1k/'
+bg_dir = './data/coco/'
+
+img_norm_cfg = dict(
+    mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225], to_rgb=True)
+train_pipeline = [  # Training data processing pipeline.
+    dict(
+        type='LoadImageFromFile',  # Load alpha matte from file.
```
Comment (innerlee) [inline configs/mattors/fba/baseline.py:36]: remove comment

File: configs/mattors/fba/baseline.py [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    ])
+# yapf:enable
+
+# runtime settings
+total_iters = 200000
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+work_dir = './work_dirs/fba'
+load_from = None
+resume_from = '/nfs/Code/mmediting/work_dirs/fba/iter_8000.pth'
```
Comment (innerlee) [inline configs/mattors/fba/baseline.py:174]: remove the hardcoded ckpt

File: configs/mattors/fba/baseline.py (hunk 1 above)
Comment (yaochaorui) [reply configs/mattors/fba/baseline.py:2]: Ok, I will delete it. I learned this from some issue in other OpenMMLab repo.

File: configs/mattors/fba/baseline.py (hunk 3 above)
Comment (yaochaorui) [reply configs/mattors/fba/baseline.py:174]: Yes.

File: configs/mattors/fba/baseline.py (hunk 2 above)
Comment (yaochaorui) [reply configs/mattors/fba/baseline.py:36]: TODO: a detailed sample config tutorial.

File: configs/mattors/fba/trial_1.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+# norm setting
+norm_cfg = dict(type='GN')
+num_groups = 32
+
+# dataset settings
+dataset_type = 'AdobeComp1kDataset'
+data_root = './data/adobe_composition-1k/'
+bg_dir = './data/coco/train2014'
+fg_dirs = [
+    data_root + 'Training_set/Adobe-licensed images/fg_extended',
```
Comment (innerlee) [inline configs/mattors/fba/trial_1.py:30]: f-string

**Pull request closed without merging**